client/src/app/filter-column/models/column.model.ts

```diff
@@ -34,71 +34,73 @@ export class ParticipantColumn {
   //mr columns
   public static MR_TYPE = new ParticipantColumn("Type", "type", "inst");
   public static MR_INSTITUTION_NAME = new ParticipantColumn("Institution Name", "name", "m");
-  public static MR_INSTITUTION_CONTACT = new ParticipantColumn("Institution Contact", "contact", "m");
+  public static MR_INSTITUTION_CONTACT = new ParticipantColumn("Institution Contact Person", "contact", "m");
   public static MR_INSTITUTION_PHONE = new ParticipantColumn("Institution Phone", "phone", "m");
   public static MR_INSTITUTION_FAX = new ParticipantColumn("Institution Fax", "fax", "m");
   public static MR_FAX_SENT = new ParticipantColumn("Initial MR Request", "faxSent", "m");
   public static MR_FAX_SENT_2 = new ParticipantColumn("Initial MR Request 2", "faxSent2", "m");
   public static MR_FAX_SENT_3 = new ParticipantColumn("Initial MR Request 3", "faxSent3", "m");
   public static MR_RECEIVED = new ParticipantColumn("Initial MR Received", "mrReceived", "m");
   public static MR_DOCUMENT = new ParticipantColumn("MR Document", "mrDocument", "m");
-  public static MR_DOCUMENT_FILES = new ParticipantColumn("MR File Names", "mrDocumentFileNames", "m");
+  public static MR_DOCUMENT_FILES = new ParticipantColumn("MR Document File Names", "mrDocumentFileNames", "m");
   public static MR_PROBLEM = new ParticipantColumn("MR Problem", "mrProblem", "m");
   public static MR_PROBLEM_TEXT = new ParticipantColumn("MR Problem Text", "mrProblemText", "m");
-  public static MR_UNABLE_TO_OBTAIN = new ParticipantColumn("MR Unable to Obtain", "unableObtain", "m");
-  public static MR_DUPLICATE = new ParticipantColumn("MR Duplicate", "duplicate", "m");
-  public static MR_INTERNATIONAL = new ParticipantColumn("MR International", "international", "m");
-  public static MR_PAPER_CR = new ParticipantColumn("MR Paper C/R Required", "crRequired", "m");
+  public static MR_UNABLE_TO_OBTAIN = new ParticipantColumn("Unable to Obtain", "unableObtain", "m");
```

Comment:
I think it is better to keep the MR in the name so it is easy to know which is which while looking at the table if both unable to obtains are selected

Reply:
true, I thought it is clear, by which dropdown shows the column, but you are right the header will not have that information anymore